src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -346,6 +346,7 @@ private void UpdateMenuButtons()
             value =>
             {
                 _showHiddenResources = value;
+                UpdateResourcesList();
```

Comment:
Should this happen after the if statement below? And if the if statement below is true, should there be a return after `AfterViewModelChangedAsync`? (because it should cause the page to redirect)

Reply:
To the first question, it doesn't really matter but feels like it's better below. I moved it. And yes, there should be - i added the return